Jieun/models/postDao.js

```diff
@@ -0,0 +1,165 @@
+const { DataSource } = require("typeorm");
+
+const dataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+dataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((err) => {
+    console.log("Error occurred during Data Source initialization", err);
+    dataSource.destroy();
+  });
+
+const createPosts = async (userId, title, content, imageUrl) => {
+  try {
+    return await dataSource.query(
+      `INSERT INTO posts (
+          user_id,
+          title,
+          content,
+          image_url
+      ) VALUES (
+          ?,
+          ?,
+          ?,
+          ?
+      )`,
+      [userId, title, content, imageUrl]
+    );
```

Comment:
```suggestion
    return dataSource.query(
      `INSERT INTO posts (
          user_id,
          title,
          content,
          image_url
      ) VALUES (
          ?,
          ?,
          ?,
          ?
      )`,
      [userId, title, content, imageUrl]
    );
```
return문에 작성되는 코드기 때문에 더이상 수행할 로직이 없고, 비동기 처리 없이 곧바로 return해도 됩니다~!